libs/scripts/delayed.js

```diff
@@ -82,7 +82,7 @@ const loadDelayed = ([
     } else {
       resolve(null);
     }
-    import('../utils/samplerum.js').then(({ sampleRUM }) => sampleRUM('cwv'));
+    import('../utils/utils.js').then(({ sampleRUM }) => sampleRUM('cwv'));
```

Comment:
This leads to a circular dependency, you can just pass sampleRUM as a dependency to loadDelayed in the utils if we stick with this approach

Reply:
done!